src/_common/ErrorMsg.js

```diff
@@ -7,9 +7,9 @@ export default class ErrorMsg extends Component {
 		shown: PropTypes.bool.isRequired,
 		text: PropTypes.string.isRequired,
 	};
-
 	render() {
 		const { shown, text } = this.props;
-		return shown ? <p className="errorfield"><M m={text} /></p> : <span />;
+        const strText = text.split(')').length > 1 ? text.split(')')[1] : text;
```

Comment:
Extract into a function in the _utils folder. Like errorToString and put this there, plust a couple of unit tests :)